# message updates the position of a certain obstacle
# model name
string model_name

# reference frame name
string frame_name

# pose
geometry_msgs/Point32 position
